app/controllers/admin/student_accounts_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+module Admin
+    class StudentAccountsController < BaseController
+      layout 'admin'
+  
+      def actions
+      end
+  
+      def get_number_of_accounts
+        days_num = get_number_of_days
+        accounts_num = query(days_num)
+        flash[:notice] = "The number of student acccounts created since July 1 is " + accounts_num.to_s
```

Comment:
Also, interesting way to do this :) (displaying a query's result as an `alert`. I would've displayed it inline like the Reports page does.

Reply:
@BryanEliDimas I used the Salesforce update as an example to create my code.